pkgs/build-support/vm/default.nix

```diff
@@ -223,11 +229,12 @@ rec {
       -nographic -no-reboot \
       -device virtio-rng-pci \
       -virtfs local,path=${storeDir},security_model=none,mount_tag=store \
+      -virtfs local,path=/build,security_model=none,mount_tag=sa \
```

Comment:
Hmm, this isn't backwards compatible and creates quite a few annoyances. Right now, it breaks disko, see https://github.com/nix-community/disko/issues/900

```
Formatting 'main.raw', fmt=raw size=10737418240
qemu-kvm: -virtfs local,path=/build,security_model=none,mount_tag=sa: cannot initialize fsdev 'sa': failed to open '/build': No such file or directory
```

The other paths in this invocation are configurable, but `/build` requires me to do a `switch_root` before. I would like to allow adding a custom prefix in front of this path. I'll submit a PR for this.

Reply:
Thank you! I don't have the bandwidth for it this weekend, can take a look at this on ~Monday if needed (please ping me here if that's the case).